### ISSUE TITLE
Mixed key case / naming convention

### ISSUE THREAD

**Issue by vreuter** [databio/dnameth_pipelines#37]
In a `stats.tsv`, some keys are snake while others are camel. Some mix these. Some of these are local to pipelines here and others come from `pypiper`. Does `pypiper` adopt a uniform convention? If not, can we change it to make that be so? Then we should match any new stat keys added here to that `pypiper` standard.

**Comment by nsheff** [comment]
I think it started using camel, but then we used snake for the `pep` standard, and the summarizer puts those together...and then chaos, probably.

**Comment by nsheff** [comment]
Ok if you look at NGStk it's exclusively using snake. So I guess that is pypiper's standard, probably to match with pepkit.

**Comment by nsheff** [comment]
but individual pipeline authors can do whatever they want... we have no control over that...

still we could document the recommendation somewhere if you like.

**Comment by vreuter** [comment]
Well I just mean that we can control it in our own pipelines, like here. It would nice also to suggest in docs, though, to cater to the OCD-inclined users among us.

**Comment by vreuter** [comment]
It looks like there's (maybe minimal, though?) mixing within `pypiper`, like in [`parse_bowtie_stats`](https://github.com/databio/pypiper/blob/master/pypiper/ngstk.py#L1691)

**Comment by nsheff** [comment]
yeah looks like it. Those aren't mine, I think that's @afrendeiro 's function. I haven't used that in any pipelines.
